package.json

```diff
@@ -77,7 +78,9 @@
   "copyright.owner": "IBM Corp.",
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-commit": "lint-staged",
```

Comment:
Cross-posting from https://github.com/strongloop/loopback-next/pull/4330#issuecomment-575685292:

 In my experience, commit hooks are not very fast, which I find annoying and thus prefer to run checks when I decide on my own. Of course, that's just me, other people may have different preferences.

and from  https://github.com/strongloop/loopback-next/pull/4382#issuecomment-586357588:

My experience with pre-commit hooks is not the best, the hook is often run in situations when I don't want it to be executed (e.g. when performing `git rebase`) and thus slowing me down.